selfservice/flow/registration/hook.go

```diff
@@ -219,6 +219,20 @@ func (e *HookExecutor) PostRegistrationHook(w http.ResponseWriter, r *http.Reque
 		return err
 	}
 
+	e.d.Audit().
+		WithRequest(r).
+		WithField("session_id", s.ID).
+		WithField("identity_id", i.ID).
+		Info("Identity authenticated successfully and was issued an Ory Kratos Session Token.")
+
+	span.AddEvent(events.NewLoginSucceeded(r.Context(), &events.LoginSucceededOpts{
```

Comment:
The code that was here before your PR is a bit misleading. While it is true that we create a session, we do not actually issue it to the user. This happens here: https://github.com/ory/kratos/blob/master/selfservice/hook/session_issuer.go#L77

I think the event should move there, because otherwise every registration event is automatically a log in event (which is not true).

Thanks!